ChangeLog.md

```diff
@@ -68,6 +68,9 @@
 * Fixed validation logic for Ipv6 IP Prefix to check for correct IPv6 prefix length
 * Added parameter set to get by subnet resource id to **Get-AzVirtualNetworkSubnetConfig** 
 * Updated description of **Location** parameter for **AzNetworkServiceTag**
+* New Generations and SKUs for VirtualNetworkGateways
```

Comment:
This should go into `src/Network/Network/ChangeLog.md`, not the one in the root

Reply:
Moved to correct changelog.